Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/clean.py

```diff
@@ -114,3 +115,51 @@ def main(cls, args, repository, **kwargs):
         for argument in args.arguments:
             result += cls.cleanup(repository, argument, remote_target=args.remote)
         return result
+
+
+class CleanPRBranches(Command):
+    name = 'clean-pr-branches'
+    help = 'Iterate through all local development branches, find matching PRs and, if those PRs are closed, delete the local branches.'
+
+    @classmethod
+    def parser(cls, parser, loggers=None):
+        parser.add_argument(
+            '--remote', dest='remote', type=str, default=None,
+            help='Specify remote to search for pull request from.',
+        )
+
+    @classmethod
+    def main(cls, args, repository, **kwargs):
+        if not repository:
+            sys.stderr.write('No repository provided\n')
+            return 1
+        if not repository.path:
+            sys.stderr.write('Cannot clean on remote repository\n')
+            return 1
+
+        rmt = repository.remote(name=args.remote)
+        if not rmt.pull_requests:
+            sys.stderr.write("'{}' cannot generate pull-requests\n".format(rmt.url))
```

Comment:
This error message would be confusing to users. User did not ask to generate a pull-request. They asked to delete local branches.

Reply:
Changing it to `'<url>'  does not have associated pull-requests`